src/pixelator/graph/backends/implementations.py

```diff
@@ -428,9 +517,118 @@ def __init__(
         """
         self._raw = raw
 
+    @staticmethod
+    def _build_plain_graph_from_edgelist(
+        df: pl.LazyFrame,
+        create_using: Union[nx.Graph, nx.MultiGraph],
+    ) -> Union[nx.Graph, nx.MultiGraph]:
+        g = nx.empty_graph(0, create_using)
+        g.add_edges_from(
+            (row[0], row[1])
+            for row in df.collect(streaming=True).iter_rows(
+                named=False, buffer_size=1000
+            )
+        )
+        return g
+
+    @staticmethod
+    def _build_graph_with_node_counts_from_edgelist(
+        df: pl.LazyFrame,
+        create_using: bool,
```

Comment:
WHat does it mean `create_using`?

Reply:
The type annotation here is a bit misleading. I've fixe that now, to make it clear that it relates to which networkx graph class should be used.